pgo-backrest/pgo-backrest.go

```diff
@@ -79,9 +79,11 @@ func main() {
 	REPO_TYPE := os.Getenv("PGBACKREST_REPO_TYPE")
 	log.Debugf("setting REPO_TYPE to %s", REPO_TYPE)
 
+	// determine the setting of PGHA_PGBACKREST_LOCAL_S3_STORAGE
+	// if there is an error, we will treat this setting as false
 	PGHA_PGBACKREST_LOCAL_S3_STORAGE, err := strconv.ParseBool(os.Getenv("PGHA_PGBACKREST_LOCAL_S3_STORAGE"))
 	if err != nil {
-		panic(err)
+		log.Warnf("%s, PGHA_PGBACKREST_LOCAL_S3_STORAGE is set to false", err)
```

Comment:
Is the warning intended to replace a perceived need for alarm (from the prior panic)? That is, does the program expect the variable to be set to something and not interpreted?

Based on the later usage of the variable, it seems that would be the place to issue a warning about no local and no S3 being a (bad?) configuration. If this warning isn't something for someone looking at logs to address, I'd recommend leaving it quiet or reduced down to INFO level.

For this use case, it seems reasonable to ignore the ParseBool error if the operating model is that boolean switches default to false unless someone affirmatively sets it to true.